### PR TITLE
LC-854 made requested changes to accessibility statement

### PR DESCRIPTION
**Pull request by jaylco**
- Changes as per Google drive document

### CODE REVIEW

**Review by neeraj-jain-cbo — CHANGES_REQUESTED** [review]
1. Please raise the PR against the idt-develop branch
2. Put the ticket number LC-854 in the beginning of the PR title.

File: src/ui/page/accessibility-statement.html [hunk 1]
```diff
@@ -40,30 +40,36 @@ <h2 class="heading-medium">Reporting accessibility problems with this website</h
...
                     for users. The Cabinet Office is not legally responsible for these learning packages meeting
                     the WCAG 2.1 AA criteria.</p>
 
+                <h2 class="heading-medium">What we're doing to improve accessibility</h2>
+                <p>We are urgently fixing content which fails to meet the Web Content Accessibility Guidelines version 2.1 AA standard. We will update this page when issues are fixed.</p>
+
                 <h2 class="heading-medium">Preparation of this accessibility statement</h2>
-                <p>This statement was prepared on 23 September 2020. It was last reviewed on 23 September 2020.</p>
-                <p>Last updated: 23 September 2020</p>
+                <p>This statement was prepared on 15 February 2021. It was last reviewed on 15 February 2021.</p>
```
Comment (neeraj-jain-cbo) [inline src/ui/page/accessibility-statement.html:71]: The last update date should be the date of the go-live.

File: src/ui/page/accessibility-statement.html [hunk 2]
```diff
@@ -40,30 +40,36 @@ <h2 class="heading-medium">Reporting accessibility problems with this website</h
...
+                <p>Last updated: 15 February 2021</p>
```
Comment (neeraj-jain-cbo) [inline src/ui/page/accessibility-statement.html:72]: The last update date should be the date of the go-live.

**Review by neeraj-jain-cbo — CHANGES_REQUESTED** [review]

File: src/ui/page/accessibility-statement.html (hunk 2 above)
Comment (jaylco) [reply src/ui/page/accessibility-statement.html]: Fixed

File: src/ui/page/accessibility-statement.html (hunk 1 above)
Comment (jaylco) [reply src/ui/page/accessibility-statement.html:71]: Do you mean last reviewed here?

**Review by neeraj-jain-cbo — APPROVED** [review]

**Pull request merged**